Module not found: can't resolve `react-flexview/lib/flexView.css`

Hi,
i want install this fantastic library, i have read the readme and when i have write in my App.js:
```js
import 'react-flexview/lib/flexView.css' // FlexView is useless without its style
```

I have recive this error:
```
./src/App.js
Module not found: Can't resolve 'react-flexview/lib/flexView.css' in ...
```

Do you have any idea for this problem?

P.S. into node_modules/react-flexview/lib/ i don't find flexView.css

Thank you so much

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
   "main": "lib",
   "scripts": {
     "test": "jest",
-    "build": "rm -rf lib && mkdir lib && tsc",
+    "build": "rm -rf lib && mkdir lib && node-sass src --importer sass-importer.js --include-path node_modules -o lib && tsc",
     "preversion": "npm run test",
     "prepublish": "npm run build",
     "start": "styleguidist server",

